GOVERNANCE.md

```diff
@@ -4,51 +4,170 @@ This document defines governance policies for the Kiali project.
 
-## Becoming a Maintainer
+Current list of maintainers (alphabetically ordered):
 
-To become a maintainer you need to demonstrate the following:
+* [aljesusg](https://github.com/aljesusg) - _Owner_
+* [hhovsepy](https://github.com/hhovsepy)
+* [israel-hdez](https://github.com/israel-hdez) - _Owner_
+* [jmazzitelli](https://github.com/jmazzitelli) - _Owner_
+* [leandroberetta](https://github.com/leandroberetta)
+* [nrfox](https://github.com/nrfox)
+* [xeviknal](https://github.com/xeviknal)
 
-  * commitment to the project
-    * participate in discussions, contributions, code reviews for 3 months or more,
-    * perform code reviews for 10 non-trivial pull requests,
-    * contribute 10 non-trivial pull requests and have them merged into project branches,
-  * ability to write high quality code,
-  * ability to collaborate with the team,
-  * understanding of team policies and processes,
-  * understanding of the project's code base and coding style.
+### Becoming a Maintainer
 
-A new maintainer must be proposed by an existing maintainer by sending a message to the
-[kiali-dev@googlegroups.com](https://groups.google.com/forum/#!forum/kiali-dev)
-mailing list containing the following information:
+To become a Maintainer you need to demonstrate the following:
 
-  * nominee's first and last name,
-  * nominee's email address and GitHub user name,
-  * an explanation of why the nominee should be a maintainer,
-  * a list of links to non-trivial pull requests (top 10) authored by the nominee.
+* commitment to the project
+  * participate in discussions, contributions, code reviews for 3 months or more,
+  * perform code reviews for 10 non-trivial pull requests,
+  * contribute 10 non-trivial pull requests and have them merged,
+* ability to write high quality code and/or documentation,
+* ability to collaborate with the team,
+* understanding of team policies and processes,
+* understanding of the project's code base, and coding and documentation style.
+
+A new maintainer must be proposed by an existing maintainer by opening a pull request
+modifying the maintainers list to add the new proposed maintainer, or via a [GitHub
+discussion](https://github.com/kiali/kiali/discussions/new) under the Governance category.
+The following information must be provided:
+
+* nominee's GitHub username,
+* an explanation of why the nominee should be a maintainer,
+* a list of links to non-trivial pull requests (top 10) authored by the nominee.
 
 Two other maintainers need to second the nomination. If no one objects in 5 working days (U.S.), the nomination is accepted.  If anyone objects or wants more information, the maintainers discuss and usually come to a consensus (within the 5 working days). If issues can't be resolved, there's a simple majority vote among current maintainers.
 
-## Changes in Maintainership
+To upgrade to _Admin_ of a Kiali repository or _Owner_ of the Kiali organization,
```

Comment:
Discussed, the "admin/owner" is a technical detail of the github repo that can be different if the project is hosted in other tool, so I don't think it deserves this wording.
To reference to have "admin privileges" in an abstract way it should be enough (those can be granted in one or another way depending of the tool).

Reply:
I think I partially agree. Decoupling from our tooling makes sense.

However, other parts already mention GitHub specific things (like GitHub discussions, issues, pull requests, teams, etc). I don't think we can fully avoid mentioning things that are specific to our tooling. So, I see that this document will anyway need to be adapted if some day we migrate to a different tool.

Anyway, I don't think we are consider moving to another tool anytime soon. So, I don't want to start thinking how to do the needed abstractions. Also, doing abstractions can add confusion so I'd lean towards being specific rather than general.

Reply:
"Admin" or "Owner" are Github terms that should not have any meaning in terms of Governance.
So they should not appear here.

Be "Maintainer" would mean to implement the privileges in the way that is required.

I'm not comfortable with those terms explictly mentioned in this document.

Reply:
I fixed. I removed the voting to become admin/owners.
There is already a sentence: "If needed, maintainers are elegible to become admins of specific repositories, or owners
of the Kiali organization.". I guess that's enough.